src/PHPUnitCommand.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Roave\NoLeaks\PHPUnit;
+
+use PHPUnit\TextUI\Command;
+use PHPUnit\TextUI\TestRunner;
+use function array_merge;
+
+final class PHPUnitCommand extends Command
+{
+    public static function main(bool $exit = true) : int
+    {
+        $command = new static();
+        return $command->run($_SERVER['argv'], $exit);
+    }
+
+    protected function handleArguments(array $argv) : void
+    {
+        parent::handleArguments($argv);
+        if (! $this->isAutoConfigureEnabled()) {
```

Comment:
I'd say that if we're running with the dedicated binary, we don't need a further switch

Reply:
Yes ! I would prefer too. Just need to ensure:

- If `phpunit.xml` has already the listener/extension registered as mentionned in the documentation -> I need to figure out a way to test that there's no "double" registration. I miss a method in PHPUnit to do it easily: `addExtension()` exists but not `getExtensions()`). Need more time to investigate.

BTW, it would fix an issue with infection (latest version run tests in random order by default) that can lead to false positive for some test-suites. Try to register no-leaks in [phpunit.xml](https://github.com/Roave/no-leaks/blob/master/phpunit.xml.dist), run infection multiple times and you'll see what I mean.

Reply:
The library is new enough to allow us to change things in a new major without affecting too many folks: let's assume that, when using this binary, we always want the extensions to be registered